src/backend/dx12/src/device.rs

```diff
@@ -653,17 +653,17 @@ impl Device {
 
     fn view_image_as_render_target(
         &self, info: ViewInfo
-    ) -> Result<d3d12::D3D12_CPU_DESCRIPTOR_HANDLE, image::ViewError> {
-        let mut pool = self.rtv_pool.lock().unwrap();
-        let handle = pool.alloc_handle();
-
+    ) -> Result<(d3d12::D3D12_CPU_DESCRIPTOR_HANDLE), image::ViewError> {
```

Comment:
Single element tuple? Does this still compile?

Reply:
this is actually the old code. fixed now regardless :)